core/src/nusi/fixed-select.tsx

```diff
@@ -17,9 +17,10 @@ import { get } from 'lodash';
 
 const { Option, OptGroup } = Select;
 const FixedSelect = React.forwardRef((props, ref) => {
+  const {options: propsOptions, ...rest} = props;
```

Comment:
prettier not working?

Reply:
There's something wrong with my prettier and it's not working.